circle.yml

```diff
@@ -0,0 +1,20 @@
+dependencies:
+  override:
+    - curl -L https://atom.io/download/deb -o atom-amd64.deb
+    - sudo dpkg --install atom-amd64.deb || true
+    - sudo apt-get update
+    - sudo apt-get -f install
+    - node --version
+    - npm --version
+    - atom --version
+    - npm prune
+    - npm install
+    - apm rebuild
+
+test:
+  override:
+    - npm test
+
+machine:
+  node:
+    version: 6.3.0
```

Comment:
Probably fine to just go with `6` here to use the latest v6.x.x version available on the CI environment.